apps/main/modules/bluetooth-spp/android/src/main/java/expo/modules/bluetoothspp/BluetoothSppModule.kt

```
@@ -0,0 +1,205 @@
+			val socket = device.createRfcommSocketToServiceRecord(SPP_UUID)
+			socket.connect()
+			return socket
+		} catch (first: IOException) {
+			// Clone printers with a broken SDP record still answer on channel 1 through the
+			// reflective constructor every printer library ends up using.
+			try {
+				val method = device.javaClass.getMethod("createRfcommSocket", Int::class.javaPrimitiveType)
+				val socket = method.invoke(device, 1) as BluetoothSocket
+				socket.connect()
+				return socket
+			} catch (second: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)